katsdpimager/frontend.py

```diff
@@ -172,6 +172,20 @@ def find_peak(image, pbeam, noise):
     return peak
 
 
+def get_totals(image_parameters, image, restoring_beam):
+    """Compute total flux density in each polarization."""
+    sums = np.nansum(image, axis=(1, 2), dtype=np.float64)   # Sum separately per polarization
+    # Area under the restoring beam. It is a Gaussian with peak of 1, and
+    # hence the area under it is 2πσ_xσ_y. The Beam class holds FWHM not
+    # standard deviations, hence the extra factor of 8*log 2.
+    beam_area = 2 * math.pi * restoring_beam.major * restoring_beam.minor / (8 * math.log(2))
```

Comment:
Are these in pixel units? I think they need to be since you are summing pixels - but please correct me if I am wrong.

Reply:
Yes, the restoring_beam is in pixels, and it gets converted to degrees when writing the FITS headers. I've added a comment.